web/src/components/TempoStackDropdown.tsx

```diff
@@ -5,12 +5,12 @@ import {
   SelectVariant,
   SelectOptionObject,
 } from '@patternfly/react-core';
-import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
 import { useTranslation } from 'react-i18next';
+import { TempoStackID } from '../../models/TempoStackID';
 
 type TempoStackDropdownProps = {
   id: string;
-  tempoStackOptions: K8sResourceCommon[];
+  tempoStackOptions: TempoStackID[];
   selectedNamespace: string | undefined;
   selectedTempoList: string | undefined;
   setTempoList: (
```

Comment:
I'm not super familiar with how we lay out our types yet, but I think that having a 1 off type in a seperate file that is just used for props doesn't make sense. I think either:

```
type TempoStackID = {
    namespace: string;
    name: string;
};

type TempoStackDropdownProps = {
  id: string;
  tempoStackOptions: TempoStackID[];
  selectedNamespace: string | undefined;
```

or 

```
type TempoStackDropdownProps = {
  id: string;
  tempoStackOptions: {namespace: string, name:string}[];
  selectedNamespace: string | undefined;
```